src/nvim/main.c

```diff
@@ -1582,6 +1582,7 @@ static void handle_tag(char *tagname)
 
     // If the user doesn't want to edit the file then we quit here.
     if (swap_exists_did_quit) {
+      ui_call_error_exit(1);
       getout(1);
```

Comment:
Can `getout` call `ui_call_error_exit` instead?

Reply:
That'll require adding another argument to `getout()` because sometimes there is an error message that the user needs to see when status is non-zero. However, when status is zero, it'll be hard to decide on the value of that argument, so I think keeping this call separate is easier .